webapp/lib/app/configurator/standard_messages/controller_messages_helper.dart

```diff
@@ -52,6 +135,23 @@ class StandardMessagesManager {
     return added;
   }
 
+  model.SuggestedReply onAddStandardMessageFromFb(model.SuggestedReply standardMessage) {
```

Comment:
I think the local/storage naming above can also be used here, so we get  addStorageMessage(s), and can work for update and remove as well.

the methods modifying the local data can use `local` instead, and so we can see the difference between them much easier too